share/spack/gitlab/cloud_pipelines/stacks/build_systems/spack.yaml

```diff
@@ -0,0 +1,61 @@
+spack:
+  view: false
+  concretization: separately
+
+  config:
+    install_tree:
+      root: /home/software/spack
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  definitions:
+    - default_specs:
+      - lz4  # MakefilePackage
+      - mpich~fortran  # AutotoolsPackage
+      - tut  # WafPackage
+      - py-setuptools  # PythonPackage
+      - openjpeg  # CMakePackage
+      - r-rcpp  # RPackage
+      - ruby-rake  # RubyPackage
+    - arch:
+      - '%gcc@7.5.0 arch=linux-ubuntu18.04-x86_64'
+
+  specs:
+    - matrix:
+        - - $default_specs
+        - - $arch
+
+  mirrors: { "mirror": "s3://spack-binaries-develop/build_systems" }
```

Comment:
Do we have to do anything to create this new mirror?

Reply:
No, nothing required as long as the bucket (in this case `spack-binaries-develop`) already exists.  You should be all set.